### PR TITLE
Sol-born battlecry, church bu- err, many other perks and emotes fixes, and rebalance. URA!!!

### PR DESCRIPTION
**Pull request by ThePainkiller**
## List of changes:

- Earth/Mars background is only applicable for Humans and Abhumans.
- Earth/Mars background now provides a perk: **_"Sol-Born Battlecry"_**. This perk grants a +10 bonus to Robustness, Toughness and Vigilance for 2 minutes, 30 minutes cooldown. This bonus **stacks** if others use the perk as well, and cannot be granted to those who are deaf or unconscious. Entirely taken off Sieg's #902 .
- This battlecry includes a voiced emote with distinction for male and female! **URAH!**
- Guts and Glory renamed to _"Will to Survive"_, still functionally identical.
- Using less bandages and ointments as a chance (10 + BIO stat) is now exclusively locked behind the _"Medical Expertise"_ perk, reserved for CBO, Doctors, Paramedics, and now given to Corpsmen as well due to the nature of the later's job.
- _"Advanced Medical Techniques"_ perk renamed to _"Advanced Surgical Techniques"_, to make better note of how it's only useful for Tissue Reconstitution Surgery, but there are plans for future reference regarding surgery to make it more useful and unique.
- Various sound emotes have had their sound code fixed as they were not playing properly, other unused sounds also added to existing emotes!
- Replaced the sound for the short whistle to express astonishment, surprise, or admiration. Use it when someone shows you an amazing gun or curio! Or something that really catches your eye...
- Various perk renames and better descriptions for their boons and curses.
- Changed text on Insight gain for more cohesion and more user-friendly tip on how to satisfy their cravings. Changed it as _"Inspiration"_ on Actor role to better reflect something being actually used for art creation.
- Many refactored perk names for code cohesion
- Ruins everything, forever, and makes everyone hate me.
	
<hr>

As a side note, non-humans can still use the `*urah` emote to scream the battlecry purely for roleplay purposes, but it will not grant them nor the ones in hearing range any bonus.

<hr>

## Changelog
:cl:
add: Added Sol-Born Battlecry perk to replace the senseless old Sol-Born perk. Grants a +10 to combat stats for 2 minutes and screams an inspiring battlecry! URA!!!
tweak: Changed certain perk names and descriptions to better convey what they do.
balance: Wasting less of a bruise pack/ointment now locked behind Medical Expertise perk. Medical Expertise perk given to Corpsmen as well. Sol-Born Battlecry perk locked to Humans and Abhumans only.
nerf: You will no longer sound like Barry White when sneezing.
fix: Fixed certain emotes not playing their assigned audio, as well as volume of some.
soundadd: Added sounds for Sol-Born Battlecry perk, changed the qwhistle emote sound for astonishment.
spellcheck: Fixed a few typos on perks descriptions
refactor: Refactored some perk names for code consistency
admin: New Sol-Born battlecry perk should be logged for admins
/:cl:

### CODE REVIEW

File: code/datums/perks/job.dm [hunk 1]
```diff
@@ -222,7 +236,7 @@
 
 /datum/perk/fast_fingers
 	name = "Fast fingers"
-	desc = "Nothing is safe around your hands. You are a true kleptomaniac. Taking items off others is without sound and prompts provided its in their pockets, hands, or on their ears. \
+	desc = "Nothing is safe around your hands. You are a true kleptomaniac. Taking items off others does no sound and prompts, provided its in their pockets, hands, or their ears. \
```
Comment (Kazkin) [inline code/datums/perks/job.dm:239]: *makes no

Comment (Kazkin) [reply code/datums/perks/job.dm:239]: *or prompts

File: code/datums/perks/job.dm [hunk 2]
```diff
@@ -241,12 +255,12 @@
...
 	name = "Expert Scavenger"
-	desc = "They say there are diamonds in the rough and you can prove it. Removing a trash pile has a chance of revealing a valuable item nobody else would find."
+	desc = "One man's trash is another man's salary. Removing a trash pile has a chance of revealing a valuable item nobody else would find."
 	//icon_state = "treasure" // https://game-icons.net
 
 /datum/perk/ass_of_concrete
-	name = "Expert Defensive Training"
-	desc = "Your position requires you to be unstoppable. Slowing down to a walking pace lets you center your gravity and few things can knock you down."
+	name = "Immovable Object"
+	desc = "Your intense training has perfected your footing, and you're an expert at holding the line. Few things can knock you off balance or push you around, as long as you remember how to anchor yourself to the floor."
```
Comment (Kazkin) [inline code/datums/perks/job.dm:263]: This does not explain how the perk works anymore, as it functions by walking, prior desc said "Slowing down to a walking pace lets" which is a direct explanation on how it works.

File: code/datums/perks/job.dm [hunk 3]
```diff
@@ -317,7 +331,7 @@
 
 /datum/perk/rezsickness/severe/fatal
 	name = "Fatal Revival Sickness"
-	desc = "You've recently died and have been brought back to life, the experience has left you utterly physically and mentally wrecked, you better find a bed to recover in..."
+	desc = "You have a (troubling) penchant for getting killed at this point. Experiencing multiple deaths and revivals has left you utterly physically and mentally wrecked, you better stay in bed for now, or you risk dying a fourth time, this time, for good."
```
Comment (Kazkin) [inline code/datums/perks/job.dm:334]: ""You have a (troubling) penchant for getting killed at this point. Experiencing multiple deaths and revivals" This doesn't really make sense given revival sickness is entirely based on toughness, the tougher you are the lest severe it is.

File: code/datums/perks/oddity.dm [hunk 4]
```diff
@@ -210,7 +210,7 @@
 
 /datum/perk/guild/blackbox_insight
 	name = "Blackbox Tinkering"
-	desc = "It's sleek contours, the expert craftsmanship, the best of hand made mechanical genius."
+	desc = "It's sleek contours, the expert craftsmanship...the best of hand-made mechanical genius."
```
Comment (Kazkin) [inline code/datums/perks/oddity.dm:213]: Missing space after the ...

File: code/datums/setup_option/backgrounds/origin.dm [hunk 5]
```diff
@@ -789,11 +789,11 @@
...
-	People from earth and mars are almost universally wealthy and worldly, with access to broad educations and understandings in philosophies that help them at every step in life. \
-	The melting pot of cultures have made Sol born extremely knowledgeable of other cultures and varied even among themselves, anyone of any race could have once been born and bred in the \
-	Sol system. However, most are still commonly human or abhuman even with the numerous exceptions. As a result of the clean living, free of pollution, drugs, or heavily processed foods \
-	your body is healthy but can't handle as many drugs as other people and you find yourself getting addicted slightly easier."
+	People from Earth and Mars are generally wealthy and worldly, with access to broad educations and understandings in philosophies that help them at every step in life. \
+	The broad contact with different cultures around the universe have made Sol born extremely knowledgeable of other races' customs and habits. However, most people born there \
+	are still mostly human or abhuman, with counted exceptions."
 
+	restricted_to_species = list(FORM_HUMAN)
```
Comment (Kazkin) [inline code/datums/setup_option/backgrounds/origin.dm:796]: I haven't tested this but this checks form, not face, its possible that abhumans who use different base forms would be barred from taking this to that over sight.

File: code/datums/perks/job.dm (hunk 3 above)
Comment (ThePainkiller) [reply code/datums/perks/job.dm:334]: You're right, I swore fatal rez sickness happened when you died with the other debuffs still active, I'll change the three to better reflect the toughness stat influence

File: code/datums/setup_option/backgrounds/origin.dm (hunk 5 above)
Comment (ThePainkiller) [reply code/datums/setup_option/backgrounds/origin.dm:796]: So, change it to `SPECIES_HUMAN` then? That way they can choose Human as species, and have their `FORM_` changed to something else and still benefit.

**Comment by Kazkin** [comment]
I specifically said I wanted sol born to be reverted and the shout perk set aside for the next update.

**Comment by ThePainkiller** [comment]
The last commit was pushed before our talk on Discord, chill.

File: code/datums/setup_option/backgrounds/origin.dm [hunk 6]
```diff
@@ -789,17 +789,17 @@
...
-	People from earth and mars are almost universally wealthy and worldly, with access to broad educations and understandings in philosophies that help them at every step in life. \
-	The melting pot of cultures have made Sol born extremely knowledgeable of other cultures and varied even among themselves, anyone of any race could have once been born and bred in the \
-	Sol system. However, most are still commonly human or abhuman even with the numerous exceptions. As a result of the clean living, free of pollution, drugs, or heavily processed foods \
-	your body is healthy but can't handle as many drugs as other people and you find yourself getting addicted slightly easier."
+	People from Earth and Mars are generally wealthy and worldly, with access to broad educations and understandings in philosophies that help them at every step in life. \
+	The broad contact with different cultures around the universe have made Sol born extremely knowledgeable of other races' customs and habits. However, most people born there \
+	are still mostly human or abhuman, with counted exceptions."
 
+	restricted_to_species = list(SPECIES_HUMAN)
```
Comment (Trilbyspaceclone) [inline code/datums/setup_option/backgrounds/origin.dm:796]:
```suggestion
	///restricted_to_species = list(SPECIES_HUMAN)
```
Do not

File: code/datums/perks/racial.dm [hunk 7]
```diff
@@ -72,9 +72,9 @@
 	user.reagents.add_reagent("adrenol", 5)
 	return ..()
 
-/datum/perk/gutsandglory
-	name = "Guts and Glory"
-	desc = "You pull what made your ancestors conquer the stars from your will, letting your body recover somewhat from any internal damage at the cost of becoming exhausted while it works."
+/datum/perk/iwillsurvive
```
Comment (Trilbyspaceclone) [inline code/datums/perks/racial.dm:75]:
```suggestion
/datum/perk/gutsandglory
```
DO NOT

File: code/datums/perks/racial.dm [hunk 8]
```diff
@@ -72,9 +72,9 @@
...
+	name = "Will to Survive"
```
Comment (Trilbyspaceclone) [inline code/datums/perks/racial.dm:76]: DO NOT
```suggestion
	name = "Guts and Glory"
```
DO NOT

File: code/datums/perks/racial.dm (hunk 7 above)
Comment (ThePainkiller) [reply code/datums/perks/racial.dm:75]: Guts and glory isn't a name that fits this perk, I agree with Sieg on this one

File: code/datums/setup_option/backgrounds/origin.dm (hunk 6 above)
Comment (ThePainkiller) [reply code/datums/setup_option/backgrounds/origin.dm:796]: I'm doing changes right now Lilly, calm down

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Lets not corrupt player data

File: code/datums/perks/racial.dm (hunk 8 above)
Comment (ThePainkiller) [reply code/datums/perks/racial.dm:76]: I will

> Lets not corrupt player data

How does this _"corrupt player data"_? This shouldn't

**Pull request merged**